gnomad_hail/utils/sample_qc.py

```diff
@@ -278,86 +278,55 @@ def assign_platform_from_pcs(
     return ht
 
 
-# TODO: This should be reviewed / merged with work from Kristen
 def infer_sex(
-        x_mt: hl.MatrixTable,  # TODO: This feels somewhat unsatisfying to provide two MTs. Maybe just reapply the QC MT filters to both (minus callrate for Y)?
-        y_mt: hl.MatrixTable,
-        platform_ht: hl.Table,
-        male_min_f_stat: float,
-        female_max_f_stat: float,
-        min_male_y_sites_called: int = 500,
-        max_y_female_call_rate: float = 0.15,
-        min_y_male_call_rate: float = 0.8
+        mt: hl.MatrixTable,
+        ploidy_ht: hl.Table,
+        f_stat_cutoff: float,
+        sites_ht: Optional[hl.Table],
+        aaf_expr: Optional[str] = None,
+        gt_expr: str = 'GT',
+        aaf_threshold: float = 0.001,
+        male_threshold: float = 0.75,
+        female_threshold: float = 0.5,
 ) -> hl.Table:
     """
     Imputes sample sex based on X-chromosome heterozygosity and Y-callrate (if Y calls are present)
 
-    :param x_mt:
-    :param y_mt:
-    :param platform_ht:
-    :param male_min_f_stat:
-    :param female_max_f_stat:
-    :param min_male_y_sites_called:
-    :param max_y_female_call_rate:
-    :param min_y_male_call_rate:
+    :param mt: Input MatrixTable
+    :param ploidy_ht: Table with imputed sex chromosome ploidies
+    :param f_stat_cutoff: f-stat to roughly divide 'XX' from 'XY' samples. Assumes XX samples are below cutoff and XY are above cutoff.
+    :param sites_ht: Optional Table to use. If present, filters input MatrixTable to sites in this Table prior to imputing sex.
+    :param aaf_expr: Optional. Name of field in input MatrixTable with alternate allele frequency.
+    :param gt_expr: Name of entry field storing the genotype. Default: 'GT'
+    :param float aaf_threshold: Minimum alternate allele frequency to be used in f-stat calculations.
+    :param float male_threshold: Threshold above which a sample will be called male.
```

Comment:
If keeping these arguments, I think it should be made clear that they are cutoffs on the F-stat  (and not ploidy)